frontend/lib/resolution/scope-queries.cpp

```diff
@@ -913,26 +913,25 @@ bool LookupHelper::doLookupInScope(const Scope* scope,
 
```

Comment:
Doesn't the "it might no longer be possible to represent the combinedFilter below as a single bitset." comment above need to be updated?